client/components/blazepress-widget/index.tsx

```diff
@@ -168,13 +168,18 @@ const BlazePressWidget = ( props: BlazePressPromotionProps ) => {
 						}
 					>
 						<Dialog
+							showCloseIcon={ true }
 							additionalOverlayClassNames="blazepress-widget"
 							isVisible={ showCancelDialog && showCancelButton }
 							buttons={ cancelDialogButtons }
 							onClose={ () => setShowCancelDialog( false ) }
 						>
 							<h1>{ translate( 'Are you sure you want to quit?' ) }</h1>
-							<p>{ translate( 'All progress in this session will be lost.' ) }</p>
+							<p>
+								{ translate(
+									'If quit, all of the work that has been done during this session will be lost.'
```

Comment:
We've received a report from a reviewer about the missing **you** pronoun before **quit**, i.e.:
`If you quit, all of the work that has been done during this session will be lost."

Reply:
I think it makes sense, I'll change it. Thanks @yuliyan

Reply:
<img width="571" alt="image" src="https://github.com/Automattic/wp-calypso/assets/43957544/1eac449d-e620-44b9-a2a7-8d1f75a479aa">
updated copy